modules/local/bbmerge/main.nf

```diff
@@ -5,6 +5,7 @@ process BBMERGE {
     input:
         tuple val(sample), path(reads)
     output:
+        tuple val(sample), path("${sample}_bbmerge_merged.fastq.gz"), emit: merged
```

Comment:
@harmonbhasin what's the thinking behind outputting the merged read file twice (once as `merged` and once in `reads`)? If you need to access this file specifically in another process, you can just access `reads[1][0]`.

Reply:
@willbradshaw I ended up fixing this. The solution you suggested works, I just had to adjust the syntax.